opm/simulators/wells/PerfData.cpp

```diff
@@ -51,6 +49,29 @@ PerfData::PerfData(std::size_t num_perf, double pressure_first_connection_, bool
     }
 }
 
+PerfData PerfData::serializationTestObject()
+{
+    PerfData result;
+    result.pressure_first_connection = 1.0;
+    result.pressure = {2.0, 3.0, 4.0};
+    result.rates = {5.0, 6.0};
+    result.phase_rates = {7.0};
+    result.solvent_rates = {8.0, 9.0};
+    result.polymer_rates = {10.0, 11.0, 12.0};
+    result.brine_rates = {13.0};
+    result.prod_index = {14.0, 15.0};
+    result.micp_rates = {16.0};
+    result.cell_index = {17, 18, 19, 20};
+    result.connection_transmissibility_factor = {21.0};
+    result.satnum_id = {22, 23};
+    result.ecl_index = {24};
+    result.water_throughput = {25.0, 26.0};
+    result.skin_pressure = {27.0, 28.0};
+    result.water_velocity = {29.0, 30.0};
```

Comment:
I think this is okay as a serialisation test.  It's not immediately clear to me if it honours all of the implicit size requirements of the `PerfData` structure and we may need to revise the setup later if, for instance, it ends up breaking some kind of checked iterator build.